index.js

```diff
@@ -1,5 +1,5 @@
 'use strict'
-const uuid = require('uuid/v4')
+const { v4: uuid } = require('uuid')
```

Comment:
There is syntax change if you compare `uuid v3.x` and the latest version. This refactor is following `uuid`'s [Quickstart](https://www.npmjs.com/package/uuid#quickstart) (with CommonJS syntax, of course)